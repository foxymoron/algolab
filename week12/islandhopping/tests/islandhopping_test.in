18
4 1 4
1 1 1
4 2 4
2 4 2
6 2 5
1 2 0 1 1
26 13 464096
24163 29842 10094 37709 26863 5748 32413 12763 5232 18895 1278 1615 19418 35741 31925 484 1500 14899 36423 12370 25748 6508 8302 37231 26931
4 4 401925
106907 244843 50174
14 27 402106
126 13058 48365 1840 67041 14647 7536 35144 35079 47369 70095 27740 34065
91 46 474001
3694 7689 7267 3191 1500 696 5591 4068 7876 4893 823 8351 3512 4553 10018 7224 10931 3303 805 6981 697 4050 1708 7012 8650 10714 9742 2307 6134 5680 3997 6542 5071 6531 1254 225 1087 2358 8432 3042 3567 4596 3762 10838 196 4823 10621 9742 4160 8960 9441 3730 6994 957 1701 5588 6695 10825 4485 5443 9728 10473 9160 9097 2917 7065 922 5269 10782 3903 6519 9602 6583 5975 681 791 31 9580 9782 1500 5005 1028 6807 828 4441 8796 2000 673 388 8351
90 36 559163
1523 9097 10463 5929 5578 1517 1795 6440 5107 808 9595 7462 8050 8872 3625 6937 2911 6015 2420 9704 7358 10783 4132 1769 5243 8674 10417 9895 1859 5717 9071 10862 7384 6113 26 2781 4454 8559 8336 423 3941 10609 9926 5567 7447 2281 9323 4883 5118 6178 2360 1873 6195 10746 5800 6132 6232 8689 3037 1714 6695 4951 6473 9138 8379 6316 10087 10326 10794 783 4601 6788 6306 2566 2736 9644 11031 8561 4813 8730 3704 7238 9949 8182 7636 10091 1347 7243 8299
49 12 383915
11245 3075 16653 14696 18261 8023 3398 10233 15894 2901 3041 5801 14690 2391 8256 7853 12918 7713 14150 6723 7987 16736 5606 97 17664 4571 7422 3372 1635 18818 15473 847 7203 15917 1446 5779 911 144 16802 959 441 8515 15071 2230 12871 142 6645 695
74 42 513065
11281 10677 9452 5573 12337 5986 6909 12962 3911 2115 12962 642 9484 10969 11968 2464 8054 4255 12246 7494 9818 4897 7688 13038 3677 7436 7396 9369 1115 9360 11836 2398 11777 8956 7905 1776 1744 12750 6177 7954 2702 6387 11143 8767 1893 4377 8282 11485 2666 10261 8496 10772 3312 13104 3703 3612 3504 12214 10090 11654 624 5771 6235 428 3987 3439 7656 1140 1371 4111 10516 10082 472
37 37 490645
8494 6110 8625 1136 25566 13041 7195 19660 10613 10049 11890 10642 26595 14023 15376 26172 1881 871 17055 7149 11407 14168 11761 4216 17507 25745 25627 3596 26142 16811 5839 5824 20497 20000 25007 14354
60 19 481527
4200 702 9964 5079 6254 3664 4643 1000 9272 16306 13872 15031 584 3364 2396 7600 1718 1229 10190 12010 10566 10687 16564 9937 11677 9013 7980 6872 11449 4985 13134 7257 6575 10780 317 11638 7217 2916 15238 7107 14099 21 670 13921 4791 16254 8681 10906 11975 14210 10958 12753 10523 15231 8139 3695 8558 279 4875
77 41 499865
945 1742 9013 2454 9090 8545 10292 12151 5788 2950 12603 3279 10624 4400 6529 1200 2839 5161 5536 3213 10186 1499 6494 4035 9421 7949 6861 4659 7618 6635 1254 7849 11341 8083 512 4285 10218 5317 10632 3453 2149 7212 9665 6794 4406 7750 9434 12780 9834 6241 2500 12168 6669 8263 11451 11736 6367 8186 8346 8011 11740 6350 3281 1744 3208 873 3161 1572 2627 10836 5665 6640 11716 8656 8253 6925
46 48 513186
11590 6132 9137 13949 5653 16788 425 4057 7885 7328 8892 18790 21214 20350 19731 8106 6715 16779 16512 558 6116 10553 14723 18769 13560 17168 6323 133 4299 16782 15664 17046 19120 16491 3874 19296 12985 313 10612 15160 7129 4379 9256 21686 11157
78 32 510787
6293 5849 7588 9230 11547 5716 9704 5444 5643 11198 705 6650 6812 8499 6 5281 6466 7681 3554 3281 1052 3846 7474 11820 7822 6529 7529 1692 10484 5702 3999 5012 9657 8041 9580 2065 9959 1989 3014 2589 8492 10110 5920 3792 5313 11528 12426 11539 818 8685 2083 2508 4917 12664 5381 2709 11827 11724 10221 6413 7978 2814 10381 6460 11468 5059 11329 2366 11561 392 12265 4799 8501 5757 88 7047 6449
69 12 504008
8223 4995 5281 3742 12515 8592 8312 312 10633 11749 1499 11412 5853 11779 807 1318 9391 9659 12919 2124 12452 14193 9759 13865 10965 13551 11965 12361 1242 5524 1587 2753 281 4315 8011 1580 11437 3548 5844 14113 11491 12426 11819 4203 3401 10464 14310 4193 7424 806 11540 2680 4553 6843 144 3399 12008 12049 9637 10634 8029 8675 4359 1260 5962 5474 13593 2170
73 48 495142
7321 2769 3474 8948 10993 7149 11491 3936 4824 4464 975 3599 11812 13528 11174 5491 10369 8237 13236 6349 11949 771 4172 4538 10846 1067 10681 10709 1368 10412 7732 6348 1281 11167 7831 8685 12896 8299 4511 6854 12011 7010 214 2729 2558 4789 8147 13287 8732 4075 11148 6757 6718 6488 15 971 5450 12089 3060 9086 6346 4094 11549 4323 11134 938 10137 2555 8025 7947 2511 7992
49 7 484692
5753 4994 16957 12954 11523 12810 17813 19025 17416 6285 5487 12300 4513 14688 4189 1204 11984 8795 8484 11203 5456 5391 13159 18446 12612 7740 17397 1607 8119 15064 17180 6757 10933 13048 7068 15721 2528 76 13011 2055 18111 12522 2690 5613 13834 17297 10104 2775
