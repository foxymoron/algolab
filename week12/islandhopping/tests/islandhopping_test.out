3 1
safe
3 3
2 462334
0 401925
0 402106
1 473805
2 558354
5 377535
2 511522
0 490645
3 479960
2 498480
0 513186
2 508917
7 495244
1 494928
8 449957
